### PR TITLE
Add notebook on TFDV

### PR DESCRIPTION
**Pull request by joppemassant**
Added the notebook from the recent TFDV chapter talk as a quick tip.

### CODE REVIEW

**Review by julienschuermans — CHANGES_REQUESTED** [review]
Great work @joppemassant, the notebook looks amazing 💯  !
Some minor remarks:

* Can you remove the references to the chapter conference talk? This is not publicly accessible anyways. The goal is for each `quick-tip` to be a completely standalone resource.

* Can you add a reference to this directory in `mlteam/quick-tips/structured_data/README.md` ? There's a list with pointers to all tips.

@others: the notebook is previewable here: https://colab.research.google.com/github/joppemassant/quick-tips/blob/structured_data/tfdv/structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb

File: structured_data/2021_06_30_tensorflow_data_validation/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+TensorFlow Data Validation (TFDV) is a tool to allow that allows you to
+detect discrepancies between datasets. Its functionality includes:
+* Visualizing your data
+* Automatically inferring a schema for your data
+* Detecting when a dataset doesn't match the schema
+* Detecting when different datasets diverge too much
+
+This chapter tip goes into the standalone version of TFDV.
+In the accompanying chapter talk (link to video), we go deeper into
+data drift & skew, and how the different divergence metrics are defined.
```
[the remark this answers is not in the document]
Comment (joppemassant) [reply structured_data/2021_06_30_tensorflow_data_validation/README.md]: Good point! Removed it!

File: structured_data/2021_06_30_tensorflow_data_validation/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# TensorFlow Data Validation
+TensorFlow Data Validation (TFDV) is a tool to allow that allows you to
+detect discrepancies between datasets. Its functionality includes:
+* Visualizing your data
+* Automatically inferring a schema for your data
+* Detecting when a dataset doesn't match the schema
+* Detecting when different datasets diverge too much
+
+This chapter tip goes into the standalone version of TFDV.
+In the accompanying chapter talk, we go deeper into
```
Comment (julienschuermans) [inline structured_data/2021_06_30_tensorflow_data_validation/README.md:10]: Can we replace this reference to the chapter talk with some URLs that *are* publicly accessible?
For non-ML6 people who are interested in more details about drift & skew metrics!

Comment (joppemassant) [reply structured_data/2021_06_30_tensorflow_data_validation/README.md:10]: Oops, seems I looked over this reference, will adapt!

File: structured_data/2021_06_30_tensorflow_data_validation/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+TFDV can also be used as part of TFX (more info here: https://www.tensorflow.org/tfx/guide#data_exploration_visualization_and_cleaning).
+
+Vertex AI has a managed version of TFDV (more info here: https://cloud.google.com/vertex-ai/docs/model-monitoring/using-model-monitoring).
+
+The notebook found in this folder goes over the different functionalities and how to use it.
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ml6team/quick-tips/blob/main/structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb)
+
+## Measuring distribution divergence
```
Comment (joppemassant) [inline structured_data/2021_06_30_tensorflow_data_validation/README.md:19]: @julienschuermans Added some information on them here

File: structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8upbZbX7gOU6"
+      },
+      "source": [
+        "Using the skew & drift comparator, you can configure your schema to allow detection of drift and skew. Both comparators are essentially the same. The skew comparator mean to be used to detect changes between training and serving statistics, and the drift comparator is used to detect changes between two different datasets in time.\n",
+        "\n",
+        "This drift and skew is measured using the following divergence metrics:\n",
```
Comment (joppemassant) [inline structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb:356]: @julienschuermans Also added it in the notebook itself

Comment (julienschuermans) [reply structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb:356]: > The skew comparator mean to be used to detect changes

typo?

Comment (joppemassant) [reply structured_data/2021_06_30_tensorflow_data_validation/tfdv_functionality_showcase.ipynb:356]: yep, fixed!

**Review by julienschuermans — APPROVED** [review]

**Comment by VHeusinkveld** [comment]
Looks good!

File: structured_data/2021_06_30_tensorflow_data_validation/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# TensorFlow Data Validation
+TensorFlow Data Validation (TFDV) is a tool to allow that allows you to
+detect discrepancies between datasets. Its functionality includes:
+* Visualizing your data.
+* Automatically inferring a schema for your data.
+* Detecting when a dataset doesn't match the schema.
+* Detecting when different datasets diverge too much.
+
+This chapter tip goes into the standalone version of TFDV.
```
Comment (VHeusinkveld) [inline structured_data/2021_06_30_tensorflow_data_validation/README.md:9]: quick tip?

**Review by VHeusinkveld — APPROVED** [review]

**Pull request merged**